framework/tests/reflection/MethodTest.java

```diff
@@ -21,35 +22,35 @@ public void real_class() {
         }
     }
 
-    public void pass1(@ReflectBottom MethodTest this) {
+    public void pass1() {
         try {
             Class<?> c = Class.forName("MethodTest$SuperClass");
             Method m = c.getMethod("getA", new Class[] {});
-            @Sibling1 Object a = m.invoke(this, (@ReflectBottom Object[]) null);
+            @Sibling1 Object a = m.invoke(superClass, (@ReflectBottom Object[]) null);
```

Comment:
There no longer is a test case that has `m.invoke(this, ....)` in a passing situation. Would having such a test continue to be useful?